bundles/org.pitest.pitclipse.listeners/META-INF/MANIFEST.MF

```diff
@@ -6,8 +6,8 @@ Bundle-Version: 2.1.2.qualifier
 Fragment-Host: org.pitest;bundle-version="1.6.8"
 Automatic-Module-Name: org.pitest.pitclipse.listeners
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
-Require-Bundle: org.pitest.pitclipse.runner;bundle-version="2.0.0"
 Bundle-ClassPath: .,
  META-INF/services/,
  src/
 Bundle-Vendor: Pitest.org
+Require-Bundle: org.pitest.pitclipse.runner
```

Comment:
If this fixes the problem you experienced then I guess it's a bug in the mechanism you've used. Could you please tell me what you were trying to do when you got the cycle (which of course does not exist)? Did you try to create an update site from Eclipse?

Reply:
I did try to create a local update site of this plug in.

This was required for me to resolve the issue.

Reply:
To that aim it's best to use the Maven Tycho build